content/en/security/threats/security_signals.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Investigate Security Signals
+kind: documentation
+disable_toc: false
+further_reading:
+  - link: "/security/default_rules/?category=cat-csm-threats#all"
+    tag: "Documentation"
+    text: "Explore CSM Threats detection rules"
+  - link: "/security/threats/workload_security_rules"
+    tag: "Documentation"
+    text: "Learn how to manage CSM Threats detection rules"
+  - link: "/security/notifications/"
+    tag: "Documentation"
+    text: "Learn more about security notifications"
+---
+
+[Cloud Security Management Threats][9] (CSM Threats) security signals are created when Datadog detects a threat based on a security rule. View, search, filter, and investigate security signals in the [Threats Explorer][4], or configure [Notification Rules][1] to send signals to third-party tools.
+
+To modify security signals, you must have the `security_monitoring_signals_write` permission. See [Role Based Access Control][3] for more information about Datadog's default roles and granular role-based access control permissions available for Cloud Security Management.
+
+{{< img src="security/cws/csm_threats_signals.png" alt="CSM Threats security signals page" width="100%">}}
+
+## Filter security signals
+
+To filter the security signals in the [Threats Explorer][4]:
+
+1. Click the **Options** button and add the column **Signal State**.
+2. Add the facet `@workflow.triage.state`. This displays signal status and enables you to sort by status in the table header.
+2. To search through signals by state, use the following search syntax `@workflow.triage.state:` and include the state (`open`, `under_review`, or `archived`) to filter on in the query.
```

Comment:
```suggestion
2. To search through signals by state, use the following search syntax `@workflow.triage.state:` and include the state (`open`, `under_review`, or `archived`) to filter on in the query.
```
I see this is elaborated here but I got a little tripped up on Step 2 , so maybe it can be re-worded into a single step to be more clear?

Reply:
Looking at this again, I don't think the first two steps are even needed anymore. The facet is now included out-of-the box. I'm going to remove those steps and rewrite this section.